### PR TITLE
Feature accessibility

### PR DESCRIPTION
**Pull request by osmanwako**
- page title  noproblem
- image text alternatives yes we have improved this issue
- Text heading yes we have added h1 on all pages
- color contrast is no problem
- size is no problem

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
       href="https://fonts.googleapis.com/css?family=Poppins"
       rel="stylesheet"
     />
-    <link rel="icon" type="image/*" href="assets/images/portfolio.png" />
+    <link rel="icon" type="image/*" href="assets/images/portfolio.png" alt="Portfolio"/>
     <link rel="stylesheet" href="./assets/style/main.css" />
     <title>Desktop version Portfolio</title>
```
Comment (souad988) [inline index.html:13]: I suggest that you use your name or profession as a title as this is your protfolio, desktop version is not referring to the project correctly

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @osmanwako , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔Good PR title
✔No linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @osmanwako, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by osmanwako** [comment]
last approved feature accessibility

**Pull request merged**